feature/community/src/main/java/com/iteneum/community/presentation/CommunityComposable.kt

```diff
@@ -0,0 +1,72 @@
+package com.iteneum.community.presentation
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.*
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import com.iteneum.community.R
+import com.iteneum.community.domain.CardMenuItem
+import com.iteneum.community.domain.CardModel
+import com.iteneum.designsystem.components.LpIconTextCard
+import com.iteneum.designsystem.theme.LeasePertTheme
+
+/**
+ * This function creates a view that displays a list of community sections.
+ * @param viewModel receives a viewModel to manage the logic of the view
+ *this view model is used to handle the loading of the components of the interface
+ * including icons, text and button click functionality.
+ *
+ * @author Andrés Ivan Medina Herrera
+ */
+@Composable
+fun CommunitylistUI() {
+    val cardWith = LeasePertTheme.sizes.extraSize124
+    val cardHeight = LeasePertTheme.sizes.extraSize128
+    val cardPadding8 = LeasePertTheme.sizes.smallerSize
+    val cardPadding15 = LeasePertTheme.sizes.smallSize
+    val cardPadding10 = LeasePertTheme.sizes.midSmallSize
+    val cardPadding20 = LeasePertTheme.sizes.minorRegularSize
+    val communityCardButtons = listOf(
+        CardModel(Icons.Outlined.Diamond, CardMenuItem.AMENITIES),
+        CardModel(Icons.Outlined.Event, CardMenuItem.EVENTS),
+        CardModel(Icons.Outlined.QuestionAnswer, CardMenuItem.COMMUNITY_WALL),
+        CardModel(Icons.Outlined.LibraryBooks, CardMenuItem.DOITYOURSELF),
+        CardModel(Icons.Outlined.Store, CardMenuItem.SERVICES)
+    )
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(top = cardPadding20, start = cardPadding10, end = cardPadding10)
+    ) {
+        Text(
+            text = "Community",
+            modifier = Modifier.padding(start = cardPadding15),
+            style = MaterialTheme.typography.titleMedium,
+        )
+        LazyVerticalGrid(
+            columns = GridCells.Fixed(2), contentPadding = PaddingValues(cardPadding8)
+        ) {
+            items(communityCardButtons) { singleCommunityCardButton ->
+                LpIconTextCard(modifier = Modifier.size(width = cardWith, height = cardHeight),
+                    icon = singleCommunityCardButton.icon,
+                    description = stringResource(
+                        id = when (singleCommunityCardButton.cardName) {
```

Comment:
Please use a Sealed class with Objects because would be easier if we decide to change the way of get the data

Reply:
Sealed class isnt part of actually more logic side than UI  please elaborate why to use it in this state, 'cos that is the plan for later

Reply:
@irv205  please confirm the use of the sealed class

Reply:
> Sealed class isnt part of actually more logic side than UI please elaborate why to use it in this state, 'cos that is the plan for later

Because you can learn how to implement the sealed class and see the pros of it if we can get the data from the server

Reply:
It would be better to implement it as @melissaquezada94 advises you, then do it, please.

Reply:
adding sealed class

 the doubts presented on whatsapp communication were not answered bud dodged, please elaborate on those comments to proceed on implementation  in a clear way 
 
 also as this is a dependency will need you to create the task with the clear description and acceptance criteria to proceed on the implementation.

Reply:
Well a sealed class can be use with a small set of cases as well an can reduce the `when` complexity and also other code complexities. 
I still think we can simplify the code and provide better flexibility in the future even if it takes more time to implement or do a research about how to use it. 
In general we want to use sealed classes instead of Enums. But if you consider to much complexity then we can leave it as enums.